src/main/java/com/api/entity/SxEngines.java

```diff
@@ -0,0 +1,46 @@
+package com.api.entity;
+
+import com.api.dto.Isp;
+import com.api.dto.ThrustStage;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class SxEngines {
+    @JsonProperty("number")
+    private int number;
```

Comment:
same here and for all Numbers (you can leave primitive only when the field is Required and never null)